REFDOC.md

```diff
@@ -0,0 +1,131 @@
+# Refactoring documentation  
+> Warning: this documentation is not yet completed. It will be completed gradually over time.  
+
+> Note: attributes are to be assumed private and methods are to be assumed public, unless noted otherwise.  
+
+> Note: attributes and methods noted as "private" are actually accessible, but the direct access of them is strongly discouraged and may results in random bugs after future refactoring.  
+
+> Note: methods return `None` unless noted otherwise.  
+
+> Warning: if the last change time indicated in this documentation is earlier than that indicated in corresponding code sections, this documentation is out of sync (and hence almost useless).
+
+# File globalvars.py  
+## Class GlobalVars.PostScanStat  
+Tracking post scanning data.  
+### Public interface  
+- `add_stat(posts_scanned, scan_time)`: Add `posts_scanned` to total numbers of posts scanned. Add `scan_time` to total time spent on scanning.
+- `get_stat()`: Get total numbers of posts scanned, total time spent on scanning and posts scanned per second. If total time spent is zero, posts scanned per second is set to `None`. Returns a tuple `(posts_scanned, scan_time, posts_per_second)`.
```

Comment:
Not a blocker, but creating excessively long lines makes it hard to edit and review this file in the future. Markdown easily lets you split source to short lines which are merged when rendering the document to HTML or other formats. Short lines which contain related phrases make for excellent diffs going forward.